src/components/artwork_filter/headline.tsx

```diff
@@ -77,4 +84,14 @@ const StyledHeadline = styled(Headline)`
   font-size: 2em;
 `
 
-export default StyledHeadline
+export default Relay.createContainer(StyledHeadline, {
+  fragments: {
+    facet: () => Relay.QL`
+      fragment on ArtworkFilterFacet {
+        ... on ArtworkFilterTag {
```

Comment:
I see the tag name here, but what about those gravity options thing? I still don't get that...